Tests/UserServiceTests/Validators/UserEmailPasswordValidatorTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using UserService.Validators;
+using DTO;
+using FluentValidation;
+using NUnit.Framework;
+using System.Linq;
+
+namespace UserServiceTests.Validators
+{
+    public class UserEmailPasswordValidatorTests
+    {
+        private UserEmailPassword emailPasswordHashEmpty = new UserEmailPassword(string.Empty, string.Empty);
```

Comment:
Why don't we initialize all these stuffs in setup?

Reply:
If I init all these stuffs in setup then all these objects will init every time I run a test, because method to be called immediately before each test is run.